daemon/exec.go

```diff
@@ -231,7 +231,12 @@ func (d *Daemon) ContainerExecStart(execName string, stdin io.ReadCloser, stdout
 }
 
 func (d *Daemon) Exec(c *Container, execConfig *execConfig, pipes *execdriver.Pipes, startCallback execdriver.StartCallback) (int, error) {
-	exitStatus, err := d.execDriver.Exec(c.command, &execConfig.ProcessConfig, pipes, startCallback)
+	pc := &execConfig.ProcessConfig
```

Comment:
Yup, I think it should be done in `ContainerExecCreate` where you create `execConfig`.